src/RelationsAns.lagda

```diff
@@ -19,6 +19,21 @@ open even
 open odd
 \end{code}
 
+* Irreflexivity
+
+\begin{code}
+open import Logic using (⊥-elim; excluded-middle; ¬_; _⊎_; inj₁; inj₂)
```

Comment:
this was the first version: https://gist.github.com/jonaprieto/80a8bdd6b200a8bb8fab865ced43bae7